src/lib.rs

```diff
@@ -2,6 +2,11 @@ use bdk::bitcoin::hashes::hex::ToHex;
 use bdk::bitcoin::secp256k1::Secp256k1;
 use bdk::bitcoin::util::psbt::PartiallySignedTransaction;
 use bdk::bitcoin::{Address, Network, Script, Txid};
+use bdk::bitcoin::util::bip32::{
+    DerivationPath as BdkDerivationPath,
+    ExtendedPrivKey as BdkExtendedPrivKey,
+    ExtendedPubKey as BdkExtendedPrivKey,
```

Comment:
Looks like a typo here:
```suggestion
    ExtendedPubKey as BdkExtendedPubKey,
```